docker-compose.yml

```diff
@@ -1,33 +0,0 @@
-version: '3.7'
-
-# Docker configuration for development
-#
-# If you need local adjustments, use a Docker compose override file.
```

Comment:
I changed my local setup to use Docker for Postgres only, and run the app without Docker. So then I realized that every local setup is different and instead of overriding this compose file it's probably just easier to let someone create their own compose file if they want.

So I still have a compose file locally, with just Postgres, but I'll leave it out of the repo.

It's also possible to run it locally with Docker without a compose file.

The server actually also uses a compose file but that one looks very different.